### ISSUE TITLE
Null package_maintainers email metadata breaks search results

### ISSUE THREAD

**Issue by adamlwgriffiths** [NixOS/nixos-search#184]
Searching for "VFIO-Tools" in 20.03 packages breaks search. This does not occur for 19.09 or unstable.

This appears to the a similar issue to #181, but based on the email metadata field.

URL in question
https://search.nixos.org/packages?query=VFIO-Tools&from=0&size=30&sort=relevance&channel=20.03

Error:
```
Bad Body
Problem with the value at json.hits.hits[19]['_source']['package_maintainers'][0].email: null Expecting a STRING
```

![image](https://user-images.githubusercontent.com/1239156/92584472-0a204980-f2d7-11ea-923f-37029620eea1.png)

Ping @garbas as he fixed #181.

**Comment by garbas** [comment]
@adamlwgriffiths thank you for reporting. I'll look at it by the end of the week.

**Comment by garbas** [comment]
fixed in #194
